### ISSUE TITLE
init.bat issues

### ISSUE THREAD

**Issue by shanff** [cmderdev/cmder#1180]
So, every time I start a "cmd" shell on Windows 10, I'm getting a  "\Windows was unexpected at this time." message.

I just downloaded 1.3.2 (full) with 161022 ConEmu with no other modifications to anything at all.  The first run ran through and setup the aliases and profile.d in the config folder and then gives me the above message.  

running the init.bat file direction also returns the message.  I think it may be %PATH% related, but I'm not really up on my Batch scripting any more.  I'm more of a Bash guy.

If I comment out the first line that turns echo off, I get the following:

```
λ init.bat

if not defined verbose-output (set verbose-output=0  )

if not defined CMDER_ROOT (for /F "delims=" %i in ("C:\Users\Shawn\Binaries\cmder\vendor\conemu-maximus5\..\..") do set "CMDER_ROOT=%~fi" )

if "r" == "\" SET "CMDER_ROOT=C:\Users\Shawn\Binaries\cmde"

if "AMD64" == "x86" (set architecture=86 )  else (set architecture=64 )

if not exist "C:\Users\Shawn\Binaries\cmder\config\settings" (
echo Generating clink initial settings in "C:\Users\Shawn\Binaries\cmder\config\settings"
 echo Additional *.lua files in "C:\Users\Shawn\Binaries\cmder\config" are loaded on startup.
)

"C:\Users\Shawn\Binaries\cmder\vendor\clink\clink_x64.exe" inject --quiet --profile "C:\Users\Shawn\Binaries\cmder\config" --scripts "C:\Users\Shawn\Binaries\cmder\vendor"

set PLINK_PROTOCOL=ssh

if not defined TERM set TERM=cygwin

if defined GIT_INSTALL_ROOT (if exist "\cmd\git.exe" (goto :FOUND_GIT ) )

setlocal enabledelayedexpansion

for /F "delims=" %F in ('where git.exe 2>nul') do @(
pushd %~dpF
 cd ..
 set "test_dir=!CD!"
 popd
 if exist "!test_dir!\cmd\git.exe" (
set "GIT_INSTALL_ROOT=!test_dir!"
 set test_dir=
 goto :FOUND_GIT
)  else (
echo Found old git version in "!test_dir!", but not using...
 set test_dir=
)
)
\Windows was unexpected at this time.
    set "PATH=!GIT_INSTALL_ROOT!\cmd;C:\Users\Shawn\Binaries\cmder\vendor\conemu-maximus5\ConEmu\Scripts;C:\Users\Shawn\Binaries\cmder\vendor\conemu-maximus5;C:\Users\Shawn\Binaries\cmder\vendor\conemu-maximus5\ConEmu;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;"C:\Program Files (x86)\Windows Kits\8.1\Windows Performance Toolkit\";"C:\Program Files (x86)\Windows Live\Shared";"C:\Program Files\Git\cmd";C:\Users\Shawn\AppData\Local\Microsoft\WindowsApps;
PS C:\Users\Shawn\AppData\Local\SquirrelTemp>;C:\Users\Shawn\AppData\Local\atom\bin;C:\Users\Shawn\AppData\Local\Microsoft\WindowsApps;"C:\Program Files (x86)\Microsoft VS Code\bin""
```

**Comment by Stanzilla** [comment]
Probably missing quotes somewhere again?

**Comment by daxgames** [comment]
I just downloaded and launched it and did not have the issue.

**Comment by daxgames** [comment]
please turn echo back off and change add
```
set verbose-output=1
```
to the top of the file rerun it and attach the output here.

**Comment by shanff** [comment]
I updated my work laptop (Windows 7) to 1.3.2 and it worked fine.  So it appears to only be on my Windows 10 laptop.  I did add the above line to the top of init.bat (above the @echo off) and got the following:

```

C:\Users\Shawn>set verbose-output=1
\Windows was unexpected at this time.
C:\Users\Shawn
```

**Comment by shanff** [comment]
I just got around to looking into my %PATH% Environmental Variables a little closer.  Looks like there were some extraneous paths and some that were left behind from other software no longer needed.  In addition, I changed c:\users\username\.. with %USERPROFILE% on a couple of them and all appears to be good in the world now.  

If anyone else has this issue, it could certainly be related to how init.bat handles bunk paths I guess.  I'm going to chalk this one up as user error.
